packs/st2ci/actions/mistral.yaml

```diff
@@ -8,11 +8,9 @@ parameters:
   test_host:
     type: string
     description: The name of the hosts where the integration test(s) will run.
-    default: None
```

Comment:
I think we always pass in this value atleast in context of the build box. Either way `None` would not be a meaningful value.